src/comments/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+import uuid
+from users.models import CustomUser
+from reviews.models import Review
+
+
+class Comment(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    review = models.ForeignKey(Review, on_delete=models.CASCADE)
+    created_by = models.ForeignKey(CustomUser, on_delete=models.CASCADE, blank=True)
+    reply_to = models.CharField(max_length=32, null=True, blank=True)
```

Comment:
what's meaning of `reply_to` field?

Reply:
reply_to sẽ chứa id của Comment cha, vì đang ở bảng Comment nên e chỉ để reply_to thay vì reply_to_comment ạ